plutus-playground-client/test/JsonEncodingTests.purs

```diff
@@ -145,9 +147,12 @@ arbitraryAssocMap genK genV = do
 
 arbitraryValue :: Gen Value
 arbitraryValue = do
-  assocMap <- arbitraryAssocMap arbitraryCurrencySymbol (arbitraryAssocMap arbitraryTokenName arbitrary)
+  assocMap <- arbitraryAssocMap arbitraryCurrencySymbol (arbitraryAssocMap arbitraryTokenName arbitraryBigInt)
   pure $ Value { getValue: assocMap }
 
+arbitraryBigInt :: Gen BigInt
```

Comment:
Does this actually generate `BigInt`s larger than `Int`?

Reply:
Oop - good spot. Yes, you're right. And fixing it turns up a larger issue with the encoding. More work needed! ⛑️

Reply:
Well spotted, thanks. And when I correct it, it throws up some missing problems. 😫